### ISSUE TITLE
scanPosition endpoint

### ISSUE THREAD

**Issue by alesgenova** [OpenChemistry/stemserver#26]
It looks to me that the `scanPosition` endpoint returns the same information (although through a different implementation) as the `frames` endpoint.
Am I missing something? @psavery

**Comment by psavery** [comment]
Yeah that is right. Originally, the `GET /stem_images/{id}/frames` was designed to return all frames, and the `GET /stem_images/{id}/frames/{scanPosition}` was designed to return just one frame.

However, because we added `limit` and `offset` to `GET /stem_images/{id}/frames`, you can now get one frame by specifying a limit of 1. So maybe `GET /stem_images/{id}/frames/{scanPosition}` is no longer needed. What do you think, @cjh1?
